testsuite/integration/elytron-oidc-client/pom.xml

```diff
@@ -46,7 +46,6 @@
         <jbossas.ts.dir>${jbossas.ts.integ.dir}/..</jbossas.ts.dir>
         <jbossas.project.dir>${jbossas.ts.dir}/..</jbossas.project.dir>
         <surefire.forked.process.timeout>3600</surefire.forked.process.timeout>
-        <wildfly.build.output.dir>build/target/${server.output.dir.prefix}-${server.output.dir.version}</wildfly.build.output.dir>
```

Comment:
@OndrejKotek This module is only available via galleon-pack, which means, it is only provisioned at the server built on build/dist.

At the moment of writing, this module is not available to other server distributions, for example, the distribution built from the ee-feature-pack module (ee-build/ee-dist). 

If your goal is to allow to supply a build output directory externally to test a custom server, then we need to ensure that these tests are not executed on the WildFly test suite when we are using ee-galleon-pack (ee-build/ee-dist), otherwise, the expectation of this Jira seems invalid.

Is your goal with this change to be able to supply an external directory built differently from what galleon-pack/ee-feature-pack modules do?